examples/v2/chat2.nim

```diff
@@ -295,6 +300,7 @@ proc readInput(wfd: AsyncFD) {.thread.} =
     let line = stdin.readLine()
     discard waitFor transp.write(line & "\r\n")
 
+{.pop.} # @TODO confutils.nim(775, 17) Error: can raise an unlisted exception: ref IOError
```

Comment:
Oh til

Reply:
Yeah, it's a life-saver, as it's a pain to get around the `confutils` issue otherwise (read: I don't know how :D)